Source/DataSourceProvider.swift

```diff
@@ -37,7 +37,9 @@ where CellFactory.Item == DataSource.Item, SupplementaryFactory.Item == DataSour
     public let supplementaryFactory: SupplementaryFactory
 
     fileprivate var bridgedDataSource: BridgedDataSource?
-
+    
+    //The data source that provides the editing functionality on the tableView
+    fileprivate var dataSourceTableEditing: TableDataSourceEditingController?
```

Comment:
Hi @jessesquires ,
Thanks for the review 😄 . The reason that I didn't make it `public` in the first place is because I wanted to constraint it (through the `extension`) so that it can be used only for a `tableView`. Since I cannot have a `stored property` in an extension I thought of setting the `dataSourceTableEditing` through a public function in the `extension`. Another option would be to have a `public computed property` in the `extension` that it's setter/getter will use either `objc runtime` or they will access directly the `fileprivate var dataSourceTableEditing`. Please let me know your thoughts on that. There might be a way that I have missed or we might don't want any of these approaches 😄

After we sort this I will proceed with applying the changes 👍